### ISSUE TITLE
rfbrowser logs errors when running robotframework with --dryrun argument

### ISSUE THREAD

**Issue by PaulBrandUWV** [MarketSquare/robotframework-browser#3915]
**Describe the bug**
Given the test teardown contains the "Go to" keyword.
When running robot with --dryrun argument.
Then rfbrowser logs errors when _start_keyword is called.
![image](https://github.com/user-attachments/assets/4d46c1f6-6e44-4fbf-89f4-f7b92c4150ce)

**To Reproduce**
Run the following test with the --dryrun argument (`robot --dryrun .\test.robot`):
```
*** Settings ***
Library    Browser

*** Test Cases ***
Test dryrun
    New browser    headless=${FALSE}
    New page    https://duck.com
    [Teardown]    Run Keyword If Test Failed    Go to    https://duck.com
```

**Expected behavior**
rfbrowser does not log errors when running with --dryrun argument.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chromium (default)
 - Version: robotframework-browser 19.0.0

**Additional context**
I think _start_keyword is called for the "Go to" keyword in the teardown, but in that case "lineno" is None.
But the "_create_keyword_call_stack_entry" method expects "lineno" to be of type int.
No errors are logged when the test have no teardown.

**Comment by aaltat** [comment]
Interesting, I think this caused by the https://github.com/MarketSquare/robotframework-browser/pull/3904 So in dry run line number is not there.

@allcontributors please add @PaulBrandUWV for bugs

**Comment by aaltat** [comment]
Fix could be easy

```
diff --git a/Browser/browser.py b/Browser/browser.py
index c8a76cec..0acf042d 100755
--- a/Browser/browser.py
+++ b/Browser/browser.py
@@ -1180,7 +1180,7 @@ def {name}(self, {", ".join(argument_names_and_default_values_texts)}):
             name or attrs.get("value", ""),
             attrs["args"],
             source,
-            attrs["lineno"],
+            attrs["lineno"] or 0,
             attrs["type"],
         )
         
 ```
 But @Snooz82  you are more familiar with this part of the RF API. Is this actually bug in RF core side and should we raise an issue about it (in any case we should handle this in our library too.) Do you see something else that could go wrong, are there some other nested structures which could go wrong?

**Comment by Snooz82** [comment]
This is actually caused by core have `None` in `lineno` which is defined as `int` in the `StartKeywordAttributes`.

```python
class StartKeywordAttributes(OptionalKeywordAttributes):
    """Attributes passed to listener v2 ``start_keyword`` method.

    See the User Guide for more information.
    """
    type: str
    kwname: str
    libname: str
    doc: str
    args: 'list[str]'
    assign: 'list[str]'
    tags: 'list[str]'
    source: str
    lineno: int
    status: str
    starttime: str
```


I will report that to core and fix it in Browser as well.

**Comment by Snooz82** [comment]
fixed in PR #3927
